joplin/js/CreateContentModal/index.js

```diff
@@ -186,6 +191,9 @@ class CreateContentModal extends Component {
                         handleTitleInputChange={this.handleTitleInputChange}
                         characterCount={this.state.titleCharacterCount}
                         maxCharacterCount={MAX_TITLE_LENGTH}
+                        departmentList={DEPARTMENT_LIST}
+                        selectedDepartment={this.state.department}
+                        handleDepartmentSelect={this.handleDepartmentSelect}
```

Comment:
for now it's just being added to the ChooseTitleStep to avoid getting into our state machine logic